src/main/java/utils/Utility.java

```diff
@@ -1,67 +1,119 @@
 package utils;
 
 public class Utility {
-    public static boolean isPalindrome(int number) {
-        StringBuilder reverseNumber = new StringBuilder();
-        String letter = String.valueOf(number);
-        for (int count = letter.length() - 1; count >= 0; count--) {
-            reverseNumber.append(letter.charAt(count));
+
+    public static class Boolean_Tests {
+        public static boolean isPalindrome(int number) {
+            StringBuilder reverseNumber = new StringBuilder();
+            String letter = String.valueOf(number);
+            for (int count = letter.length() - 1; count >= 0; count--) {
+                reverseNumber.append(letter.charAt(count));
+            }
+            //return reverseNumber.toString().equals(letter);
+            return true;
+        }
+
+        public static boolean oddOrEven(int nmb) {
+            return nmb % 2 == 0;
+        }
+
+        public static boolean isMultiple(int nmb1, int nmb2) {
```

Comment:
is important to use variable names that are clear too.
in this simple case would be better to use as in mathematics 'x' and 'y'
it is preferred to don't use abbreviations and numbers in the variable names it rarely allowed
so often when you have a class **UserRepository** the name of the reference is **userRepository**
abbreviations are used when the names are very long and cannot be replaces by a shorter analogy
Change the parameter and variables names following the conventions
https://www.oracle.com/technetwork/java/codeconventions-135099.html

Reply:
changed it to 'number'